packages/connectivity_plus/connectivity_plus/darwin/Classes/PathMonitorConnectivityProvider.swift

```diff
@@ -1,30 +1,34 @@
 import Foundation
 import Network
 
-@available(iOS 12, *)
+@available(iOS 12, macOS 10.14, *)
```

Comment:
This makes no sense as with recent changes plugins already have MacOS 10.14 as min deployment target. Same applies to iOS 12.

Reply:
I've removed that part to reflect this, especially since installation of this plugin is indeed confined to using a pod.